core/storage/trie/impl/trie_error.cpp

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "trie_error.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::storage::trie, TrieError, e) {
+  using kagome::storage::trie::TrieError;
+  switch (e) {
+    case TrieError::SUCCESS:
+      return "success";
+    case TrieError::NO_VALUE:
+      return "no value";
```

Comment:
Please elaborate on the error message, it seems a bit unclear.

Reply:
fixed